client/my-sites/plans-grid/hooks/npm-ready/data-store/use-grid-plans.tsx

```diff
@@ -308,7 +307,7 @@ const useGridPlans = ( {
 		term,
 		intent,
 	} );
-	const planUpgradeability = usePlanUpgradeabilityCheck?.( {
+	const plansAvailabilityForPurchase = useCheckPlanAvailabilityForPurchase?.( {
```

Comment:
It's not letting me leave a comment on L339, but I also needed to update `planUpgradeability?.[ planSlug ]` to `useCheckPlanAvailabilityForPurchase?.[ planSlug ]` to get the local dev build running 🙂

Reply:
Thanks @jeyip ! Looks like I focused almost entirely on the tests passing 😆

Reply:
BTW good point here whether we need the full check `isInSignup || plansAvailabilityForPurchase?.[ planSlug ]`... You'd think `plansAvailabilityForPurchase` should be enough. 

Something for a follow-up perhaps. I'd rather not make any more changes right now, as you both tested enough of this already @jeyip / @southp 🙇

Reply:
Added a big todo in the code for now. Might turn into an issue, but it's also something we'll be seeing in front of us a lot, being the main hook that compiles everything. We ought to revisit.